integration_test/testgenerated/errors/errors_test.go

```diff
@@ -158,3 +189,38 @@ func TestError_IsErrorType(t *testing.T) {
 	assert.False(t, api.IsMyInternal(fmt.Errorf("error")))
 	assert.False(t, api.IsMyInternal(errors.NewInternal()))
 }
+
+func TestError_Cause(t *testing.T) {
+	err := api.NewMyNotFound(api.Basic{}, []int{}, "", "", nil)
+	assert.Nil(t, err.Cause())
+
+	wrappedErr := werror.Error("cause")
+	err = api.WrapWithMyNotFound(wrappedErr, api.Basic{}, []int{}, "", "", nil)
+	assert.Equal(t, wrappedErr, err.Cause())
+}
+
+func TestError_StackTrace(t *testing.T) {
+	err := api.NewMyNotFound(api.Basic{}, []int{}, "", "", nil)
+	stack := fmt.Sprintf("%+v", err.StackTrace())
+	assert.Contains(t, stack, "TestError_StackTrace")
+}
+
+func TestError_Format(t *testing.T) {
+	const safeParamValue = "typeFoo"
+
+	err := api.NewMyNotFound(api.Basic{}, []int{}, safeParamValue, "", nil)
+	printedErr := fmt.Sprintf("%+v", err)
+
+	// Message
+	assert.Contains(t, printedErr, err.Message())
+
+	// Safe params
+	for k := range err.SafeParams() {
+		assert.Contains(t, printedErr, k)
+	}
+	assert.Contains(t, printedErr, err.InstanceID().String())
+	assert.Contains(t, printedErr, safeParamValue)
+
+	// Stack trace
+	assert.Contains(t, printedErr, "TestError_Format")
```

Comment:
when we fix the stackframe skipping let's assert that we do not have those function names in here

Reply:
Done.